worker/include/RTC/WebRtcServer.hpp

```diff
@@ -98,7 +98,7 @@ namespace RTC
 
 	public:
 		// Passed by argument.
-		const std::string id;
+		std::string id;
```

Comment:
Why this?

Reply:
The clang-tidy rule that suggested to remove it is [this](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/avoid-const-or-ref-data-members.html). There is further reading [here](https://quuxplusone.github.io/blog/2022/01/23/dont-const-all-the-things/#data-members-never-const), which argues why having const members is never a good idea.

Having said this, this change should go along making `id` private and adding a `GetId()` API, which is how we usually do it rather than having a public `id`. 

Are we fine making `id` private and adding a `GetId()` method?

Reply:
I'm ok with GetId() but those links basically also recommend to avoid local scope constants and many things we are doing on purpose to, ironically, make clang-tidy happy.

Reply:
There is no clang-tidy warning after the change. This rule is very specific about not making members private. Otherwise you can not, for example, have this:

```c++
struct Employee {
    std::string name;
    const std::string id;
};

Employee e1 = {"Alice", "12345"};
Employee e2 = {"Bob", "24601"};
e1 = e2; // oops
std::swap(e1, e2); // oops
```

I'm going to create `GetId()`